Source/Core/RequestScheduler.js

```diff
@@ -129,41 +256,93 @@ define([
      * }
      *
      */
-    RequestScheduler.throttleRequest = function(url, requestFunction) {
+    RequestScheduler.throttleRequest = function(request) {
```

Comment:
I'm putting too much thought into this, but perhaps better semantics are `schedule()` (today's `throttleRequest`) and `request()` (today's `request`).